backend/smb/connpool.go

```diff
@@ -235,15 +236,18 @@ func (f *Fs) drainPool(ctx context.Context) (err error) {
 	if len(f.pool) != 0 {
 		fs.Debugf(f, "Closing %d unused connections", len(f.pool))
 	}
+
+	g, _ := errgroup.WithContext(ctx)
 	for i, c := range f.pool {
-		if !c.closed() {
-			cErr := c.close()
-			if cErr != nil {
-				err = cErr
+		g.Go(func() (err error) {
```

Comment:
I think this needs

    i, c := i, c

for old go 1.21 and earlier....

Scratch that the mimimum required version is now go 1.22 so it is fine :-)